app/api/v2/handlers/operation_api.py

```diff
@@ -226,8 +227,8 @@ async def get_operation_link(self, request: web.Request):
                               'description': 'UUID of the link object to retrieve results of.'
                           }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
-    @aiohttp_apispec.response_schema(LinkSchema(partial=True),
-                                     description='Contains a result string for the link requested.')
+    @aiohttp_apispec.response_schema(LinkResultSchema(partial=True),
```

Comment:
Is there a case where we'd want `partial=True`?

```suggestion
    @aiohttp_apispec.response_schema(LinkResultSchema(),
```

Reply:
I moved this to the nested LinkSchema inside of LinkResultSchema